<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="image_btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>130</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>original image</string>
    </property>
    <property name="iconSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="mark_btn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>130</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>mark</string>
    </property>
    <property name="iconSize">
     <size>
      <width>130</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>inpainting operation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inpaint_ns_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Navier-Stoke Method</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inpaint_fmm_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Fast-Matching Method</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>960</width>
      <height>550</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="image">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>960</width>
       <height>550</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1043</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="select_image_menu"/>
    <addaction name="save_image_menu"/>
    <addaction name="separator"/>
    <addaction name="load_mark_menu"/>
    <addaction name="save_mark_menu"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="select_image_menu">
   <property name="text">
    <string>Select Image</string>
   </property>
  </action>
  <action name="save_image_menu">
   <property name="text">
    <string>Save Result</string>
   </property>
  </action>
  <action name="actionCreate_New_Mark">
   <property name="text">
    <string>Create New Mark</string>
   </property>
  </action>
  <action name="load_mark_menu">
   <property name="text">
    <string>Load Mark</string>
   </property>
  </action>
  <action name="save_mark_menu">
   <property name="text">
    <string>Save Mark</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave_Old_Mark_and_Create">
   <property name="text">
    <string>Save Old Mark and Create</string>
   </property>
  </action>
  <action name="actionCreate_Mark">
   <property name="text">
    <string>Create Mark</string>
   </property>
  </action>
  <action name="actionClear_Mark_Field">
   <property name="text">
    <string>Clear Mark Field</string>
   </property>
  </action>
  <action name="actionAlgorith_1">
   <property name="text">
    <string>Algorithm 1</string>
   </property>
  </action>
  <action name="actionAlgorithm_2">
   <property name="text">
    <string>Algorithm 2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
